boards/nxp/imx95_evk/doc/index.rst

```diff
@@ -115,88 +115,139 @@ Programming and Debugging (A55)
 
 .. zephyr:board-supported-runners::
 
-Use this configuration to run basic Zephyr applications and kernel tests,
-for example, with the :zephyr:code-sample:`synchronization` sample:
+There are multiple methods to program and run Zephyr on the A55 core:
+
+Option 1. Boot Zephyr by Using SPSDK Runner
+===========================================
+
+SPSDK runner leverages SPSDK tools (https://spsdk.readthedocs.io), it builds an
+bootable flash image ``flash.bin`` which includes all necessary firmware components,
+such as ELE+V2X firmware, System Manager, TCM OEI, TF-A images etc. Using west flash
+command will download the boot image flash.bin to DDR memory, SD card or eMMC flash.
+By using flash.bin, as no U-Boot image is available, so TF-A will boot up Zephyr on
+the first Cortex-A55 Core directly.
+
+In order to use SPSDK runner, it requires fetching binary blobs, which can be achieved
+by running the following command:
+
+.. code-block:: console
 
-1. Build and run the Non-SMP application
+   west blobs fetch hal_nxp
+
+.. note::
+
+   It is recommended running the command above after :file:`west update`.
+
+SPSDK runner is enabled by configure item ``CONFIG_BOARD_NXP_SPSDK_IMAGE``, currently
```

Comment:
use proper syntax for all Kconfig items i.e.
```
:kconfig:option:`CONFIG_BOARD_NXP_SPSDK_IMAGE`
```

Reply:
Thanks @nordicjm 
Updated.